src/Authentication/Authentication/Cmdlets/ConnectMgGraph.cs

```diff
@@ -13,18 +13,18 @@ namespace Microsoft.Graph.PowerShell.Authentication.Cmdlets
     using System.Net;
     using System.Globalization;
     using System.Collections;
+    using System.Security.Cryptography.X509Certificates;
 
-    using Microsoft.Identity.Client;
+    using Identity.Client;
 
-    using Microsoft.Graph.Auth;
-    using Microsoft.Graph.PowerShell.Authentication.Helpers;
-    using Microsoft.Graph.PowerShell.Authentication.Models;
+    using Auth;
```

Comment:
This can lead to confusion since one won't know where the namespace comes from. For instance, Auth (.NET auth providers) and Helpers (PS auth module) are from 2 different libraries. Also, all other class in the SDK use fully qualified namespaces.

Reply:
Okay, I'll revert back to the older format